apps/firecrawler/.env.deprecated

```
@@ -0,0 +1,292 @@
+# Production: Recommended for high-security environments
+FIRECRAWLER_ENABLE_REQUEST_SIGNING=false
+
+# Request signature secret (if signing enabled)
+# Generate with: openssl rand -base64 32
+FIRECRAWLER_SIGNATURE_SECRET=
+
+# Advanced Configuration
+# ===========================================
+
+# Custom user agent for web requests
+FIRECRAWLER_USER_AGENT="Firecrawl-MCP/0.1.0"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)